learn-data-structures-by-building-a-shopping-cart/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>freeCodeCamp Shopping cart</title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1 class="title">Desserts page</h1>
+    </header>
+    <main>
+      <button id="cart-btn" type="button" class="btn">
+        <span id="show-hide-cart">Show</span> Cart
+      </button>
+      <div id="shopping-cart-container">
```

Comment:
One thing I noticed throughout the code is that cart elements are sometimes referred to as "cart" and other times as "shopping cart".

Either one works, though I lean more towards using "cart" everywhere except the `title` element since it's a bit shorter and seems more common on major sites.

Reply:
Sure, that makes sense. 
I will standardize that to reference cart 👍